cypress/support/step-definitions/common-steps.js

```diff
@@ -342,6 +342,10 @@ Then('closeIcon is not visible', () => {
   closeIconButton().should('not.exist');
 });
 
+Then('closeIcon is not visible in NoIFrame', () => {
+  closeIconButtonNoIFrame().should('not.exist');
```

Comment:
Can you change the name to `closeIconButton` and if there is a `closeIconButton` for Iframe then change it as `closeIconButtonIframe`, as we decided
```suggestion
  closeIconButton().should('not.exist');
```